frontend/src/components/home/CardView.js

```diff
@@ -0,0 +1,37 @@
+import React from "react";
+import { Card, Row, Col } from "antd";
+
+const CardView = props => {
+    const cards = props.cardData.map(cardItem => {
```

Comment:
Another possibility is to rename `cardData` to `data`, `items` or `projects`. I think calling it `cardData` is redundant because you pass the props to `CardView` anyway.